lib/utils/openBrowser.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/* eslint-disable no-console */
+
+const opn = require('./open');
+const chalk = require('chalk');
+const isDockerContainer = require('is-docker');
+const BbPromise = require('bluebird');
+
+function displayManualOpenMessage(url, err) {
+  // https://github.com/sindresorhus/log-symbols
+  console.log('---------------------------');
```

Comment:
Why we do not rely here on serverless logger? Even if we don't like to use `cli.log` we may rel on `cli.consoleLog`.

It'll clearly indicate the intention of generating user facing CLI output.

Direct use of `console.log` is more for temporary debug output (which we prevent from committing in via lint rules)